python/lsst/dax/imgserv/vo/imageSODA.py

```diff
@@ -265,8 +290,8 @@ def get_brect(self, params: dict) -> afw_image:
 
         Returns
         -------
-        image : afw_image
-
+        cutout : `lsst.afw.image`
+i
```

Comment:
I'm not sure I get this part... you've got different functions for the shapes, but they all have the same body, and don't check that the shape is what you're passing in.  Maybe just use one and drop this layer of abstraction?

Reply:
Should seriously consider dropping this layer - once we know for sure we won't implement a GENERIC (non-LSST) version of SODA - for now the interface (abstract) requires this layer.